### PR TITLE
Migrate Auction app to Metaphysics V2

### PR DESCRIPTION
**Pull request by dblandin**
Migrate Auction app to use Metaphysics V2.

### CODE REVIEW

File: src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx [hunk 1]
```diff
@@ -48,36 +48,36 @@ const setupTestEnv = ({
       />
     ),
     query: graphql`
-      query ConfirmBidValidTestQuery @raw_response_type {
```
Comment (alloy) [inline src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx]: @dblandin Why did you remove this here?

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:
